client/packages/requisitions/src/ResponseRequisition/DetailView/ReponseStats/ResponseStoreStats.tsx

```diff
@@ -44,31 +44,37 @@ export const ResponseStoreStats: React.FC<ResponseStoreStatsProps> = ({
           paddingBottom: 2,
         }}
       >
-        <Box
-          display="flex"
-          alignItems="flex-start"
-          width={predictedStockPercent}
-        >
-          <ValueBar
-            value={stockOnHand}
-            total={predictedStockLevels}
-            label={t('label.stock-on-hand')}
-            colour="gray.dark"
-            startDivider
-          />
-          <ValueBar
-            value={incomingStock}
-            total={predictedStockLevels}
-            label={t('label.incoming-stock')}
-            colour="gray.main"
-          />
-          <ValueBar
-            value={stockOnOrder}
-            total={predictedStockLevels}
-            label={t('label.stock-on-order')}
-            colour="gray.light"
-          />
-        </Box>
+        {stockOnHand === 0 && incomingStock === 0 && stockOnOrder === 0 ? (
+          <Typography fontSize={14}>
+            {t('messages.no-stock-available')}
+          </Typography>
```

Comment:
```suggestion
          <Typography fontSize={14} style={{ textAlign: 'center' }}>
            ⓘ
            <span style={{ fontStyle: 'italic', paddingLeft: 4 }}>
              {t('messages.no-stock-available')}
            </span>
          </Typography>
```

though I also like having 'currently' as part of the message. It says to me that this is a temporary problem 🤔